packages/slate/src/models/node.js

```diff
@@ -1351,6 +1351,41 @@ class Node {
     return path
   }
 
+  /**
+   * refind the path of node if path is changed
+   *
+   * @param {Array} path
+   * @param {Node|String} node
+   * @return {Array}
+   */
+
+  refindPath(path, node) {
+    const key = (typeof node === 'string') ? node : node.key
+    const tryNode = this.getDescendantAtPath(path)
+    if (tryNode && tryNode.key === key) {
+      return path
+    }
+    return this.getPath(key)
+  }
+
+  /**
+   *
+   * refind the node with the same node.key after change
+   *
+   * @param {Array} path
+   * @param {Node|String} node
+   * @return {Node|Void}
+   */
+
+  refindNode(path, node) {
```

Comment:
Can you change these methods to be called `refresh` instead of `refind`? I want to avoid confusion with the existing `find` methods which take iterators instead.